opty/direct_collocation.py

```diff
@@ -686,6 +686,35 @@ def parse_free(self, free):
 
         return parse_free(free, n, q, N, variable_duration)
 
+    def time_vector(self, t0=0.0, solution=None):
```

Comment:
```suggestion
    def time_vector(self, solution=None, start_time=0.0):
```

I think people will rarely use the start time kwarg, so make it second. And rename to have more informative variable name.

Reply:
I now wonder, whether we should give t0 at all: I would not know how to tell opty not to start at t= = 0.0

Reply:
We can leave it off and add it later if we ever think we need it.

Reply:
> We can leave it off and add it later if we ever think we need it.

I left it in. Do you want me to remove it?